cmd/cc-uploader/main.go

```diff
@@ -189,16 +131,87 @@ func initializeServer(logger lager.Logger, uploaderConfig config.UploaderConfig,
 			tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
 		}
 
-		server = &http.Server{
-			Addr:      uploaderConfig.MutualTLS.ListenAddress,
-			Handler:   ccUploaderHandler,
-			TLSConfig: clientTLSConfig,
-		}
-		return server, http_server.NewTLSServer(uploaderConfig.MutualTLS.ListenAddress, ccUploaderHandler, clientTLSConfig)
+		return http_server.NewTLSServer(uploaderConfig.MutualTLS.ListenAddress, ccUploaderHandler, clientTLSConfig)
```

Comment:
ifrit runs counter to idiomatic go. This seems to deepen the dependency on ifrit.

The community needs to decide, whether we want to add more ifrit, or go more towards idiomatic go, channels, contexts and goroutines, instead of subprocesses and OS signals.

Reply:
In capi open office hour (10. September 2025) there were no complaints. But if we wish to have a discussion on it, there was the proposal to open an issue for it.

Reply:
Not from my side. I said my piece.